docs/source/simulation_study_results.rst

```diff
@@ -0,0 +1,317 @@
+Simulation studies
+==================
+
+This document is currently a work in progress. The results documented
+here originate from Gerbrich Ferdinands’ master’s thesis which can be
+found `here
+<https://github.com/GerbrichFerdinands/asreview-thesis/blob/master/manuscript/manuscript/Ferdinands%2C-G---MSBBSS.pdf/>`_.
+
+
+To provide insight in how much screening effort ASReview could
+potentially save, seven ASReview models were simulated on six existing
+systematic review datasets. In short, for all six datasets ASReview
+could have saved at least 60% of screening effort (e.g. WSS95% was <
```

Comment:
This doesn't sound right. I would also add the '@', because I think that's how it's presented in literature. Correct me if I'm wrong. Either way, unless I'm sorely mistaken, higher values of WSS@95 denote better results. (So, should be > 60% I think).

Reply:
No you are correct, my bad!